cli/src/klio_cli/commands/job/utils/templates/MANIFEST.in.tpl

```diff
@@ -1 +1,2 @@
 include job-requirements.txt
+include klio-job-run-effective.yaml
```

Comment:
Is this indeed needed? (referring to our own [docs](https://docs.klio.io/en/latest/faqs/migrate_from_fnapi.html#new-manifest-in) - not including `yaml` files weren't needed before; just non-Python files needed to install the package)

Reply:
ugh, apparently it isn't.  I tested this a bunch but now I think there was an issue with my test setup.  I'll remove.